adapters/invibes/invibes.go

```diff
@@ -230,6 +230,7 @@ func (a *InvibesAdapter) makeRequest(invibesParams InvibesInternalParams, reqInf
 		Uri:     url,
 		Headers: headers,
 		Body:    body,
+		ImpIDs:  openrtb_ext.GetImpIDs(request.Imp),
```

Comment:
This is not an accurate list of the impression ids for this adapter. `MakeRequests` will skip invalid impressions which are still included in `request.Imp`. Instead, this should iterate over the values of `invibesParams.BidParams.Properties` and extract the `ImpID` to a slice.

Reply:
Yes. you are correct.
Reading impIds from bidParamsProps